_posts/2014-04-05-overview.md

```diff
@@ -5,30 +5,37 @@ permalink: overview
 github: "https://github.com/ember-cli/ember-cli.github.io/blob/master/_posts/2014-04-05-overview.md"
 ---
 
-Ember CLI is the [Ember.js](http://emberjs.com) command line utility that
-provides a fast [Broccoli](https://github.com/broccolijs/broccoli#broccoli)-powered
-asset pipeline, a strong conventional project structure, and a powerful [addon
-system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints)
-for extension. [Get started.](/user-guide/#getting-started)
+Ember CLI is the official command line utility used to create, build, serve, 
+and test [Ember.js](http://emberjs.com) apps and addons.
+
+It provides a consistent project structure, a powerful [addon
+system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints), and
+a fast [Broccoli](https://github.com/broccolijs/broccoli#broccoli)-powered
+asset pipeline.
+Whether you are looking for a zero-config experience or the ability to make your own customizations, Ember CLI has the tools you need.
+
+There are thousands of JavaScript libraries made specifically for Ember.
+We call them addons.
```

Comment:
I think it might be important to include that addons are just npm packages themselves; we didn't create our own custom weird thing.